lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -117,13 +121,18 @@
         )
         logger.debug(". . . Retrieved the ALB private key from SSM Parameter Store.")
 
+        # Include all VA CA certificates in the default SSL environment.
+        # ssl_context = ssl.create_default_context(capath=CA_PATH)
+        # TODO - This is a workaround.  The capath approach doesn't seem to load anything.
```

Comment:
Since you are adding a TODO here, is there an associated ticket?

Reply:
Good call

Reply:
I added a reference to #1063.